framework/src/seclink/seclink.c

```diff
@@ -95,6 +95,20 @@ extern int sl_post_msg(int fd, int cmd, unsigned long arg);
 		free(handle);							\
 	} while (0)
 
+#define NSPERSEC 1000000000
+static uint32_t _calc_elasped(struct timespec *start, struct timespec *end)
```

Comment:
Please add `#include <time.h>` for usages of structures and functions like struct timespec, clock_gettime